server/tests/api/check-params/config.ts

```diff
@@ -47,6 +47,11 @@ describe('Test config API validators', function () {
         css: 'body { background-color: red; }'
       }
     },
+    federation: {
+      videos: {
+        federateUnlisted: false
```

Comment:
Do you want admins to update this setting on the web interface? If no, you can remove this